client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-credentials/index.tsx

```diff
@@ -115,6 +115,8 @@ const SiteMigrationCredentials: Step< {
 			from_url: fromUrl,
 			blog_url: siteSlug,
 		} );
+		// Reset the site in the state to ensure the correct overview screen is shown.
+		siteId && dispatch( resetSite( siteId ) );
```

Comment:
We should wait for the send ticket call to be solved before continuing with the cache clean and the redirect.